corehq/apps/users/models.py

```diff
@@ -724,16 +724,29 @@ def migrate_eula(cls, data):
     def is_eula_signed(self, version=CURRENT_VERSION):
         if self.is_superuser:
             return True
-        for eula in self.eulas:
-            if eula.version == version:
-                return eula.signed
+        current_domain = getattr(self, 'current_domain', None)
+        current_eula = self.eula
+        if current_eula.version == version:
+            if toggles.FORCE_ANNUAL_TOS.enabled(current_domain):
+                elapsed = datetime.now() - current_eula.date if current_eula.date else 365
```

Comment:
Nitpick: it'd be a little clearer to add a `if not current_eula.date return False` rather than using the dummy 365 value when there's no date

Reply:
yeah, I agree. I was on the fence about that. will update.